src/app/main-nav/main-nav/main-nav.component.html

```diff
@@ -55,14 +56,27 @@
           {{ organization.name }}
         </button>
         <mat-divider></mat-divider>
-        <a
-          mat-menu-item
-          id="create-new-link"
-          routerLink="/organizations/new"
-          i18n
-          >Create New Organization</a
-        >
+        <ng-container *ngIf="canCreateOrg$ | async; else creationDisabled">
+          <a
+            mat-menu-item
+            id="create-new-link"
```

Comment:
@Nanyte25 I think this is the change that is causing the build failure.. let me check from upstream how they are handling the e2e